docs/core/compatibility/core-libraries/9.0/unsafeaccessor-generics.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Breaking change: Altered UnsafeAccessor support for non-open generics"
+description: Learn about the .NET 9 breaking change in core .NET libraries where support for UnsafeAccessor with non-open generics has been altered.
+ms.date: 07/29/2024
+---
+# Altered UnsafeAccessor support for non-open generics
```

Comment:
The C# spec doesn't use the term "non-open". Generics are either "open" (having at least one type parameter), or "closed" (all type parameters are specified by an argument).

I'd prefer not adding a new term that isn't defined.